### ISSUE TITLE
Babel performance

### ISSUE THREAD

**Issue by buckelieg** [jarslab/babel-maven-plugin#9]
I am using babel 6.26.0 which seems to do the work.
But it runs very slow.
I propose to implement ["compact"](https://mariusschulz.com/blog/speeding-up-babel-transpilation-with-compact-mode) plugin setting to improve performance.

**Comment by jonaskoperdraat** [comment]
@buckelieg I've added a PR (#12) which addresses the plugins performance. Using a different strategy, I was able to significantly improve the performance.

I haven't tried using the 'compact' setting, and looking at the provided link, that seems like a fix for certain buggy Babel releases. Anyway, I think instead of adding various properties, the plugin stands to benefit from a method of providing babel with a more elaborate configuration, as opposed to only specifying presets. Still thinking about how to implement that, though 😉
